src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -25,6 +25,29 @@
  */
 final class CachePoolClearCommand extends ContainerAwareCommand
```

Comment:
`getContainer()` is protected so the deprecation would be triggered only if called from a subclass, but this is final :)